cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/index.js

```diff
@@ -277,3 +303,20 @@ export default class ProfilesCreateView extends Component {
     );
   }
 }
+
+const mapStateToProps = (state) => {
+  return {
+    loading: state.loading,
+    provisionerJsonSpecMap: state.map,
+    selectedProvisioner: state.selectedProvisioner
+  };
+};
+const ConnectedProfilesCreateView = connect(mapStateToProps)(ProfilesCreateView);
+
+export default function ProfilesCreateViewFn() {
+  return (
+    <Provider store={ProvisionerInfoStore}>
+      <ConnectedProfilesCreateView />
```

Comment:
You need to pass `props.location` to this component, otherwise `this.props.location` will be undefined inside the connected component